test/local_links_manager_api_test.rb

```diff
@@ -66,6 +66,30 @@
         assert_equal expected_response, response.to_hash
       end
 
+      it "returns the local authority details only if no link present for westminster" do
```

Comment:
Nitpick: I think we can delete this test - we're not really testing logic for Westminster AFAIK, we're just checking that the provided `snac` code comes back in the response.

We could achieve the same thing by updating the test at line 46 instead, swapping `blackburn` for `westminster` and passing the custom `snac` code 🤔